### ISSUE TITLE
can't access rclone folder

### ISSUE THREAD

**Issue by racine-p-a** [gotson/komga#811]
### Steps to reproduce

First of all, many thanks for this awesome project.

- Install komga via docker `gotson/komga` `sudo docker create --name=komga --network=nginx --user 0:0 -p 4005:8080 --mount type=bind,source=/path/tokomga/config,target=/config --mount type=bind,source=/path/to/komga/data,target=/data --restart unless-stopped  gotson/komga:latest`
- place comics in a simple folder and try to create a library --> SUCCESS ! :D
- create a rclone folder from a bigger FTP drive `rclone mount my_remote:/files/my_folder /home/user/local/path/to/data/ --no-checksum --use-server-modtime --no-gzip-encoding --no-update-modtime --vfs-cache-mode full &` --> SUCCESS
- place same comics from usual library to the distant ftp folder
- check if the rclone worked correctly on the local machine `ls /home/user/local/path/to/data/` displays the list of the comics --> SUCCESS
- go to komga interface and create library using this new folder (with the option about the timestamps)
- create a new library using the new rclone folder
- scan --> FAIL (0.148.3) no file displayed and no errors in logs
- scan --> FAIL (v0.152.0-master) no file displayed and no errors in logs

### Expected behavior

I should see the comics when i add their folder in a new library (+scan of course)

### Actual behavior

Nothing appears/is fount.

### Logs

InfoCollection=true, importComicInfoReadList=true, importEpubBook=true, importEpubSeries=true, importMylarSeries=true, importLocalArtwork=true, importBarcodeIsbn=false, scanForceModifiedTime=true, scanDeep=false, repairExtensions=false, convertToCbz=false, emptyTrashAfterScan=false, seriesCover=FIRST, hashFiles=true, hashPages=false, analyzeDimensions=true, unavailableDate=null, id=07V0KFY9P612G, createdDate=2022-02-18T21:20:10, lastModifiedDate=2022-02-18T21:20:10)
2022-02-18 21:20:18.292  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.k.domain.service.FileSystemScanner   : Scanning folder: /data/sacha/test3
2022-02-18 21:20:18.292  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.k.domain.service.FileSystemScanner   : Supported extensions: [cbz, zip, cbr, rar, pdf, epub]
2022-02-18 21:20:18.293  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.k.domain.service.FileSystemScanner   : Excluded patterns: [#recycle, @eaDir]
2022-02-18 21:20:18.293  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.k.domain.service.FileSystemScanner   : Force directory modified time: true
2022-02-18 21:20:18.294  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.k.domain.service.FileSystemScanner   : Scanned 0 series, 0 books, and 0 sidecars in 308.573us
2022-02-18 21:20:18.296  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.k.d.service.LibraryContentLifecycle  : Scan returned no series, soft deleting all existing series
2022-02-18 21:20:18.297  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.k.domain.service.SeriesLifecycle     : Soft delete series: []
2022-02-18 21:20:18.298  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.komga.domain.service.BookLifecycle   : Soft delete books: []
2022-02-18 21:20:18.305  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.k.d.s.SeriesCollectionLifecycle      : Deleting empty collections
2022-02-18 21:20:18.308  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.k.domain.service.ReadListLifecycle   : Deleting empty read lists
2022-02-18 21:20:18.311  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.k.d.service.LibraryContentLifecycle  : Library updated in 18.862065ms
2022-02-18 21:20:18.316  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.komga.application.tasks.TaskEmitter  : Sending task: FindBooksToConvert(libraryId='07V0KFY9P612G', priority='0')
2022-02-18 21:20:18.323  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.komga.application.tasks.TaskEmitter  : Sending task: FindBooksWithMissingPageHash(libraryId='07V0KFY9P612G', priority='0')
2022-02-18 21:20:18.323  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.komga.application.tasks.TaskEmitter  : Sending task: FindDuplicatePagesToDelete(libraryId='07V0KFY9P612G', priority='0')
2022-02-18 21:20:18.326  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.komga.application.tasks.TaskHandler  : Task ScanLibrary(libraryId='07V0KFY9P612G', priority='8') executed in 34.894535ms
2022-02-18 21:20:18.333  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.komga.application.tasks.TaskHandler  : Executing task: FindBooksToConvert(libraryId='07V0KFY9P612G', priority='0')
2022-02-18 21:20:18.334  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.komga.domain.service.BookConverter   : CBZ conversion is not enabled, skipping
2022-02-18 21:20:18.334  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.komga.application.tasks.TaskHandler  : Task FindBooksToConvert(libraryId='07V0KFY9P612G', priority='0') executed in 1.098084ms
2022-02-18 21:20:18.337  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.komga.application.tasks.TaskHandler  : Executing task: FindBooksWithMissingPageHash(libraryId='07V0KFY9P612G', priority='0')
2022-02-18 21:20:18.338  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.k.domain.service.PageHashLifecycle   : Page hashing is not enabled, skipping
2022-02-18 21:20:18.338  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.komga.application.tasks.TaskHandler  : Task FindBooksWithMissingPageHash(libraryId='07V0KFY9P612G', priority='0') executed in 939.936us
2022-02-18 21:20:18.339  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.komga.application.tasks.TaskHandler  : Executing task: FindDuplicatePagesToDelete(libraryId='07V0KFY9P612G', priority='0')
2022-02-18 21:20:18.341  INFO 1 --- [org.springframework.jms.JmsListenerEndpointContainer#1-1] o.g.komga.application.tasks.TaskHandler  : Task FindDuplicatePagesToDelete(libraryId='07V0KFY9P612G', priority='0') executed in 1.715746ms

### Komga version

0.148.3 and v0.152.0-master (via docker)

### Operating system

Ubuntu 20.04

### Other details

Your ptoject is awesome :)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have checked the [FAQ](https://komga.org/faq/).
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by DieselTech** [comment]
Read the rclone guide written on the site:

https://komga.org/guides/gdrive.html

**Comment by racine-p-a** [comment]
I readt it before even starting using rclone

**Comment by DieselTech** [comment]
Okay but your rclone create command doesn't use the recommenced values that the page has listed.

**Comment by racine-p-a** [comment]
Understood, i did it : 
`rclone mount DD:/files/test4 /my/path/to/data/test4/ --no-checksum --use-server-modtime --no-gzip-encoding --no-update-modtime --no-seek --modify-window 2m --allow-other --allow-non-empty --dir-cache-time 30m --cache-read-retries 15 --cache-db-purge --timeout 30m --vfs-cache-mode full --vfs-read-chunk-size 2M --vfs-read-chunk-size-limit 5M --vfs-cache-max-age 30m --attr-timeout 20s --poll-interval 9m --vfs-cache-poll-interval 10m &`

This pointed an error : 
`2022/02/18 22:31:47 mount helper error: fusermount: option allow_other only allowed if 'user_allow_other' is set in /etc/fuse.conf
2022/02/18 22:31:47 Fatal error: failed to mount FUSE fs: fusermount: exit status 1`

So i modified `/etc/fuse.conf` accordingly. I retried and still nothing in the interface. I checked the logs in `/config/logs/komga.log` and nothing weird :/

**Comment by racine-p-a** [comment]
Oh and thanks for your help fast as lightning, @DieselTech :D

**Comment by gotson** [comment]
Nothing related to Komga, get your rclone working properly first.

**Comment by gotson** [comment]
For that kind of thing, better come and ask on Discord for help.

**Comment by racine-p-a** [comment]
Hello @gotson 

It seems to not be related to rclone : i tried the exact same thing with `curlftpfs`. And the result is exactly the same. BUT ! I fount something weirdly interesting : 
- i mounted the ftp folder using `curlftpfs` --> SUCCESS
- i browsed the mounted folder and made `ls`, `mkdir`, etc... --> SUCCESS
- I went to komga interface, i added a new library, it asked me which folder, i browsed the folders - INSIDE the mounted folder! - so it means that komga has access to the mounted folder and all its subfolders ! But when i select the subfolder containing the comics, komga founds nothing (even after a scan) --> FAIL

--> The comics used to test this were fount without any problem when using a local (normal) folder
--> I made a `chmod -R 777` on the folder to be sure that it is not a permission problem.
--> Still no information in the logs

I am writing here despite you closed the issue just in case my tests might help someone in the future. As you said, i am going to ask on discord hoping someone had the same problem in the past.

Regards
